js/libs/form.js

```diff
@@ -1032,17 +1032,7 @@ Fliplet().then(function() {
       mounted: function() {
         var $vm = this;
 
-        this.saveProgress = debounce(function() {
-          var progress = {};
-
-          $vm.fields.forEach(function(field) {
-            if (field.saveProgress !== false && field.enabled) {
-              progress[field.name] = field.value;
-            }
-          });
-
-          localStorage.setItem(progressKey, JSON.stringify(progress));
-        }, saveDelay);
+        this.saveProgressed = debounce(saveProgress(), saveDelay);
```

Comment:
@NikitaJadhavXoriant Have you confirmed this works? It looks like your syntax is wrong.

```suggestion
        this.saveProgressed = debounce(saveProgress, saveDelay);
```

Reply:
Hi @tonytlwu, its flow is working fine, we checked, still should i go ahead and remove paranthesis?

Reply:
@NikitaJadhavXoriant Yes please. See https://lodash.com/docs/4.17.15#debounce that `_.debounce()` takes a function as the first parameter. What you've current got here is not passing a function as the first parameter.

<img width="471" alt="image" src="https://github.com/Fliplet/fliplet-widget-form-builder/assets/290733/a2b2379a-fcc8-439e-94a4-db019fe5a5d7">

You should see that if the first parameter isn't a function, it generates an error.